addons/l10n_syscohada/data/account.account.template.csv

```diff
@@ -1014,51 +1014,51 @@
+"pcg_833","CHARGES LIEES AUX OPERATIONS DE RESTRUCTURATION","833","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_834","PERTES SUR CREANCES H.A.O.","834","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_835","DONS ET LIBERALITES ACCORDES","835","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_836","ABANDONS DE CREANCES CONSENTIS","836","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_837","CHARGES LIEES AUX OPERATIONS DE LIQUIDATION","837","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_839","CHARGES  POUR DEPRECIATIONS ET PROVISIONS POUR RISQUES A COURT TERME H.A.O.","839","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_841","PRODUITS H.A.O CONSTATES","841","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_843","PRODUITS LIES AUX OPERATIONS DE RESTRUCTURATION","843","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_844","INDEMNITES ET SUBVENTIONS H.A.O. (entité agricole)","844","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_845","DONS ET LIBERALITES OBTENUS","845","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_846","ABANDONS DE CREANCES OBTENUS","846","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_847","PRODUITS LIES AUX OPERATIONS DE LIQUIDATION","847","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_848","TRANSFERTS DE CHARGES H.A.O","848","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_849","REPRISES DE CHARGES POUR DEPRECIATIONS ET PROVISIONS POUR RISQUES A COURT TERME H.A.O.","849","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
+"pcg_851","DOTATIONS AUX PROVISIONS RÉGLEMENTÉES","851","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_852","DOTATIONS AUX AMORTISSEMENTS H.A.O.","852","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_853","DOTATIONS AUX DÉPRÉCIATION H.A.O.","853","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_854","DOTATIONS AUX PROVISIONS POUR RISQUES ET CHARGES H.A.O.","854","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_858","AUTRES DOTATIONS H.A.O.","858","account.data_account_type_expenses","l10n_syscohada.syscohada_chart_template","False"
+"pcg_861","REPRISES DE PROVISIONS REGLEMENTEESREPRISES DE PROVISIONS RÉGLEMENTÉES","861","account.data_account_type_revenue","l10n_syscohada.syscohada_chart_template","False"
```

Comment:
Something is off here with the name

Reply:
I've check in the official COA, it seems good, what do you find off ?

Reply:
@malb-odoo Is it normal that the name is double?

Reply:
Ah didn't saw the last one ahah, yep will changed that